getting-started/get-the-sdk.md

```diff
@@ -53,55 +53,8 @@ You should see a pop-up similar to the following \(image below shows iOS\):
 For React Native, it is recommended to first install [Node.js](https://nodejs.org) to download packages from [npm](https://npmjs.com). See this [tutorial](https://facebook.github.io/react-native/docs/getting-started) for additional instructions on getting started with React Native applications
```

Comment:
- Please change to "For React Native, we recommend that you first install..."
- Please reword as follows:
"for additional instructions on getting started with React Native applications, see this [tutorial](https://facebook.github.io/react-native/docs/getting-started)."